build/platform-android.mk

```diff
@@ -47,6 +47,38 @@ CXXFLAGS += -fno-rtti -fno-exceptions
 LDFLAGS += --sysroot=$(SYSROOT)
 SHLDFLAGS = -Wl,--no-undefined -Wl,-z,relro -Wl,-z,now -Wl,-soname,lib$(PROJECT_NAME).so
 
+ifeq ($(NDK_TOOLCHAIN_VERSION), clang)
+  HOST_OS = $(shell uname -s | tr A-Z a-z)
+  CC = $(NDKROOT)/toolchains/llvm/prebuilt/$(HOST_OS)-x86_64/bin/clang
+  CXX = $(NDKROOT)/toolchains/llvm/prebuilt/$(HOST_OS)-x86_64/bin/clang++
+  AR = $(NDKROOT)/toolchains/llvm/prebuilt/$(HOST_OS)-x86_64/bin/llvm-ar
+  ifeq ($(ARCH), arm)
+  CFLAGS += -target armv7-none-linux-androideabi
+  CFLAGS += --sysroot=$(NDKROOT)/sysroot -I$(NDKROOT)/sysroot/usr/include/arm-linux-androideabi/
+  LDFLAGS += -target armv7a-none-linux-androideabi -isysroot $(SYSROOT)
+  LDFLAGS += -B $(NDKROOT)/toolchains/arm-linux-androideabi-4.9/prebuilt/$(HOST_OS)-x86_64/bin/arm-linux-androideabi-
+  LDFLAGS += -L$(NDKROOT)/toolchains/arm-linux-androideabi-4.9/prebuilt/$(HOST_OS)-x86_64/lib/gcc/arm-linux-androideabi/4.9.x
+  else ifeq ($(ARCH), arm64)
+  CFLAGS += -target aarch64-none-linux-androideabi
```

Comment:
This shouldn't be `androideabi` but plain `android` here

Reply:
will fix this. thanks!